__tests__/app.test.js

```diff
@@ -140,4 +141,61 @@ describe("GET /api", () => {
         });
     });
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Returned set of comments should be in an array", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+            expect(Array.isArray(res.body.comments)).toBe(true);
+        });
+    })
+
+    test("200: Comments should have correct properties", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+            res.body.comments.forEach((comment) => {
```

Comment:
If you are forEaching through an array you need to do a length check, if an empty array comes back this test will pass.